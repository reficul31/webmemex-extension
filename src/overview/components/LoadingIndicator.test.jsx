/* eslint-env jest */

import React from 'react'
import LoadingIndicator from 'src/overview/components/LoadingIndicator'
import { shallow } from 'enzyme'

describe('LoadingIndicator', () => {
    test('should test against the given snapshot', () => {
        const tree = shallow(
            <LoadingIndicator />
        )
        expect(tree).toMatchSnapshot()
    })
})
