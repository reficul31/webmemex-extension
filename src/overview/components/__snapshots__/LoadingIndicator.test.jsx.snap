// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingIndicator should test against the given snapshot 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="container"
>
    <span
        className="dotone dot"
    />
    <span
        className="dottwo dot"
    />
    <span
        className="dotthree dot"
    />
</div>,
  "nodes": Array [
    <div
      className="container"
>
      <span
            className="dotone dot"
      />
      <span
            className="dottwo dot"
      />
      <span
            className="dotthree dot"
      />
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <LoadingIndicator />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="container"
>
          <span
                    className="dotone dot"
          />
          <span
                    className="dottwo dot"
          />
          <span
                    className="dotthree dot"
          />
</div>,
        "_debugID": 2,
        "_renderedOutput": <div
          className="container"
>
          <span
                    className="dotone dot"
          />
          <span
                    className="dottwo dot"
          />
          <span
                    className="dotthree dot"
          />
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <LoadingIndicator />,
}
`;
